### PR TITLE
Resolve CS1591.

### PR DESCRIPTION
**Pull request by yukun-dong**
Some ObjectTypeProperties' descriptions are empty and this causes CS1591(missing XML comment for publicly visible type or member). For above situations, this PR uses declaration names as descriptions.
# Description
<i>Add your description here!</i>

# Checklist

To ensure a quick review and merge, please ensure:
- [ ] The PR has a understandable title and description explaining the _why_ and _what_.
- [ ] The PR is opened in draft if not ready for review yet.
   - If opened in draft, please allocate sufficient time (24 hours) after moving out of draft for review
- [ ] The branch is recent enough to not have merge conflicts upon creation.

# Ready to Land?
- [ ] Build is completely green
   - Submissions with test failures require tracking issue and approval of a CODEOWNER
- [ ] At least one +1 review by a CODEOWNER
- [ ] All -1 reviews are confirmed resolved by the reviewer 
   - Override/Marking reviews stale must be discussed with CODEOWNERS first

### CODE REVIEW

**Review by ArcturusZhang — APPROVED** [review]
I am not sure if we need to expand this to all of the object models - for instance, models might also not have their description. But this LGTM

**Review by archerzz — APPROVED** [review]
LGTM. Any better idea of the default description template? :)

File: samples/AppConfiguration/Generated/Models/Key.cs [hunk 1]
```diff
@@ -16,12 +16,13 @@ internal Key()
...
 
         /// <summary> Initializes a new instance of Key. </summary>
-        /// <param name="name"></param>
+        /// <param name="name"> The Name. </param>
         internal Key(string name)
         {
             Name = name;
         }
 
+        /// <summary> The Name. </summary>
```
Comment (m-nash) [inline samples/AppConfiguration/Generated/Models/Key.cs:25]: this should be "Gets the [lower case property name]" unless its not readonly then "Gets or sets the [lower case property name]"

**Review by m-nash — CHANGES_REQUESTED** [review]

**Review by ArcturusZhang — APPROVED** [review]

File: samples/AppConfiguration/Generated/Models/Key.cs (hunk 1 above)
Comment (yukun-dong) [reply samples/AppConfiguration/Generated/Models/Key.cs]: @m-nash Hi Michael, I have modified it according to your request, please take a look 😃  .

**Pull request merged**